lib/shopifyHmac/extractHmacParameter.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+function extractHmacParameter({ query }) {
+  if (!query.hmac) {
+    throw new Error(
+      "Could not extract Hmac parameter: no Hmac parameter found in query."
```

Comment:
should capitalize HMAC consistently
```suggestion
      "Could not extract HMAC parameter: no HMAC parameter found in query."
```